vsintegration/tests/unittests/Tests.OpenDeclarationsGetter.fs

```diff
@@ -0,0 +1,159 @@
+﻿[<NUnit.Framework.TestFixture>][<NUnit.Framework.Parallelizable>]
+module Tests.OpenDeclarationsGetter
```

Comment:
First goal was to get the old PR running on the newest master. That's why all code has been taken over. The cleanup begins now, I already asked for someone taking a look on this and telling me whether the test system here is still in use.